packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2542,6 +2554,9 @@ export const DEFAULT_COMPOSITE_ICONS: {
     SurveyStarIconFilled: React_2.JSX.Element;
 };
 
+// @public (undocumented)
+export type DeletedChatMessage = Omit<ChatMessage_2, 'message' | 'metadata'>;
```

Comment:
Unsure if this is the best payload to transmit for the event, but it creates the closest type to the other events

Reply:
Is the `undocumented` in the comment `// @public (undocumented)` expected?  
Where does the `DeletedChatMessage` being defined?

Reply:
Removing this, not needed